src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -127,20 +128,54 @@ class AddDependencySearchResult extends PureComponent<Props> {
 
     const downloadNumColor = getColorForDownloadNumber(hit.downloadsLast30Days);
 
+    // We have to use a custom highlight
+    // to unescape the html
+    const CustomHighlight = connectHighlight(
+      ({ highlight, attribute, hit, highlightProperty }) => {
+        const highlights = highlight({
+          highlightProperty: '_highlightResult',
+          attribute,
+          hit,
+        });
+
+        return highlights.map((part, i) => {
+          // Run the parser on each of the parts
+          const unescaped = (new DOMParser().parseFromString(
+            `<span>${part.value}</span>`,
+            'text/html'
+          ): any).body.textContent;
```

Comment:
I have the feeling this could be useful elsewhere. Let's move it to a function in `utils.js`:

```js
// A comment explaining what this does, with some examples, would be great
export const safeEscapeString = (val: string) => (
  new DOMParser()... // rest of stuff here
);
```